catalog/task/run-file-updates/0.2/run-file-updates.yaml

```diff
@@ -0,0 +1,65 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: run-file-updates
+  labels:
+    app.kubernetes.io/version: "0.2"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to run file updates
+  params:
+    - name: jsonKey
+      type: string
+      description: The json key containing the file updates
+      default: ".spec.extraData.fileUpdates[]"
+    - name: fileUpdatesPath
+      type: string
+      description: The path to the file containing the file updates
+    - name: snapshotPath
+      type: string
+      description: Path to the JSON string of the Snapshot spec in the data workspace
+      default: "snapshot_spec.json"
+    - name: request
+      type: string
+      description: Name of the request
+      default: "file-updates"
+    - name: synchronously
+      type: string
+      description: Whether to run synchronously or not
+      default: "true"
+  workspaces:
+    - name: data
+      description: Workspace where the file updates to apply are defined
+  steps:
+    - name: run-script
+      image: quay.io/hacbs-release/release-utils:a918b3a7cbd40add5955a2f1acb4c3dbb29c44e9
+      script: |
+        #!/bin/sh
+        set -ex
+
+        # Read JSON file into a variable
+        jsonData=$(cat $(workspaces.data.path)/$(params.fileUpdatesPath))
+
+        # Extract the key from the JSON file
+        fileUpdates=$(echo "${jsonData}" | jq -rc "$(params.jsonKey)")
```

Comment:
why not just cat the file here and `|` instead of creating a variable just to echo it one time?

Reply:
I agree. Also, I would recommend iterating over indexes of the array (and `cat` the original file each time we need to get a piece of data).

Reply:
same as the next comment. I just added the `update-paths` command. I was not willing to rewrite this task, but ok. I will do it.

Reply:
BTW, I guess my main "issue" with this - why I suggested going over the index - is that I don't think it's good to store tons of data in env vars. I would even have thought that there was a limit, but apparently not: https://stackoverflow.com/questions/5076283/shell-variable-capacity

So maybe it's just me and it's completely fine? :)

(Even the new version is still "guilty" of this, but again, maybe it's just me, so I'm fine with the current version.)